planning/behavior_path_planner_common/package.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>behavior_path_planner_common</name>
+  <version>0.1.0</version>
+  <description>The behavior_path_planner_common package</description>
+
+  <!-- turn signal decider -->
+  <maintainer email="zulfaqar.azmi@tier4.jp">Zulfaqar Azmi</maintainer>
+  <maintainer email="mamoru.sobue@tier4.jp">Mamoru Sobue</maintainer>
+  <maintainer email="satoshi.ota@tier4.jp">Satoshi Ota</maintainer>
+
+  <!-- drivable area generation-->
+  <maintainer email="maxime.clement@tier4.jp">Maxime CLEMENT</maintainer>
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <maintainer email="satoshi.ota@tier4.jp">Satoshi Ota</maintainer>
+
+  <!-- maintainer -->
+  <maintainer email="tomoya.kimura@tier4.jp">Tomoya Kimura</maintainer>
+  <maintainer email="shumpei.wakabayashi@tier4.jp">Shumpei Wakabayashi</maintainer>
+  <maintainer email="takayuki.murooka@tier4.jp">Takayuki Murooka</maintainer>
+  <maintainer email="kosuke.takeuchi@tier4.jp">Kosuke Takeuchi</maintainer>
+  <maintainer email="fumiya.watanabe@tier4.jp">Fumiya Watanabe</maintainer>
+  <maintainer email="takamasa.horibe@tier4.jp">Takamasa Horibe</maintainer>
+  <maintainer email="zulfaqar.azmi@tier4.jp">Zulfaqar Azmi</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <author email="taiki.tanaka@tier4.jp">Taiki Tanaka</author>
+  <author email="takamasa.horibe@tier4.jp">Takamasa Horibe</author>
+  <author email="satoshi.ota@tier4.jp">Satoshi Ota</author>
+  <author email="fumiya.watanabe@tier4.jp">Fumiya Watanabe</author>
+  <author email="zulfaqar.azmi@tier4.jp">Zulfaqar Azmi</author>
+  <author email="kosuke.takeuchi@tier4.jp">Kosuke Takeuchi</author>
+  <author email="yutaka.shimizu@tier4.jp">Yutaka Shimizu</author>
```

Comment:
:cry: 

```suggestion
```

Reply:
I think it is ok to keep him as author not maintainer. What do you think?